core/libraries/form_sections/strategies/validation/EE_Equal_To_Validation_Strategy.strategy.php

```diff
@@ -0,0 +1,61 @@
+<?php
+use EventEspresso\core\domain\services\factories\EmailAddressFactory;
+use EventEspresso\core\domain\services\validation\email\EmailValidationException;
+use EventEspresso\core\exceptions\InvalidDataTypeException;
+use EventEspresso\core\exceptions\InvalidInterfaceException;
+
+/**
+ * Class EE_Email_Validation_Strategy
+ *
+ * @package               Event Espresso
+ * @subpackage            core
+ * @author                Mike Nelson
+ * @since                 4.6
+ */
+class EE_Equal_To_Validation_Strategy extends EE_Text_Validation_Strategy
+{
+
+    protected $_compare_to = null;
+
+
+    /**
+     * @param string               $validation_error_message
+     */
+    public function __construct($validation_error_message = '', $compare_to)
+    {
+        if (! $validation_error_message) {
+            $validation_error_message = esc_html__('Confirm email doesn\'t match.', 'event_espresso');
```

Comment:
since you've named this validation strategy in an abstract enough way that it could apply to anything (which is great), 
let's change this error message to also be generic... maybe something like:

```suggestion
            $validation_error_message = esc_html__('Fields do not match.', 'event_espresso');
```

or if you want to get fancy, you could require the "presentation" label for the `$compare_to` field as a parameter and then `sprintf() ` that into the error message.